codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -110,6 +105,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
         }
     }
 
+    private boolean isNotWebSocketPath(FullHttpRequest req) {
+        return checkStartsWith ? !req.uri().startsWith(websocketPath) : !req.uri().equals(websocketPath);
```

Comment:
I don't think this is much different from a readability standpoint. The reason why I included it was because there was some performance benefit claimed by this PR. I don't think it is necessary ... just a curiosity if part of the justification for this PR is performance.